core/no_certificate_validation.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+use rustls::internal::msgs::handshake::DigitallySignedStruct;
+use rustls::Certificate;
+use rustls::HandshakeSignatureValid;
+use rustls::RootCertStore;
+use rustls::ServerCertVerified;
+use rustls::ServerCertVerifier;
+use rustls::TLSError;
+use webpki::DNSNameRef;
+
+pub struct NoCertificateVerification {
+  excluded: Vec<String>,
+}
+
+impl NoCertificateVerification {
+  pub fn new(excluded: Vec<String>) -> Self {
+    Self { excluded }
+  }
+}
```

Comment:
So I guess this has been put here because it's used in both `extensions/fetch` and `extensions/net`? If so we might need to just copy-paste this bit of code to both crates. `deno_core` is completely agnostic of TLS and shouldn't contain this code.

Reply:
@bartlomieju Some background: I could not find a better place to put it. First MVP which I had locally I had this struct in the `extensions/net` and then in `extensions/fetch` I referenced it.

Reply:
That's understandable and I think copy-pasting is the best solution to this situation.